<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkDemonsRegistrationViewControls</class>
 <widget class="QWidget" name="QmitkDemonsRegistrationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>287</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkDemonsRegistrationViewControls</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QComboBox" name="m_RegistrationSelection">
     <item>
      <property name="text">
       <string>Demons Registration</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Symmetric Forces Demons Registration</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="histogramMatchingBox_2">
     <property name="title">
      <string>Histogram Matching</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="_2">
        <item>
         <layout class="QVBoxLayout" name="_3">
          <item>
           <widget class="QLabel" name="useHistogramMatching">
            <property name="text">
             <string>Use Histogram Matching:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="numberOfHistogramLevels">
            <property name="text">
             <string>Number of Histogram Levels:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="numberOfMatchPoints">
            <property name="text">
             <string>Number of Match Points:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="thresholdAtMeanIntensity">
            <property name="text">
             <string>Threshold at Mean Intensity:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="_4">
          <item>
           <widget class="QCheckBox" name="m_UseHistogramMatching">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_NumberOfHistogramLevels">
            <property name="text">
             <string>1024</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_NumberOfMatchPoints">
            <property name="text">
             <string>7</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_ThresholdAtMeanIntensity">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="demonsRegistrationBox_2">
     <property name="title">
      <string>Demons Registration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="_5">
        <item>
         <layout class="QVBoxLayout" name="_6">
          <item>
           <widget class="QLabel" name="iterationsLabel">
            <property name="text">
             <string>Number of Iterations:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="standardDeviationLabel">
            <property name="text">
             <string>Standard Deviation:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="_7">
          <item>
           <widget class="QLineEdit" name="m_Iterations">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>100</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_StandardDeviation">
            <property name="text">
             <string>1.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <includes>
  <include location="local">mitkDataTreeNode.h</include>
  <include location="local">mitkDemonsRegistration.h</include>
  <include location="local">mitkHistogramMatching.h</include>
  <include location="local">mitkSymmetricForcesDemonsRegistration.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
